src/main/kotlin/com/terraformation/backend/funder/model/FundingEntityModel.kt

```diff
@@ -11,7 +12,7 @@ data class FundingEntityModel(
     val name: String,
     val createdTime: Instant,
     val modifiedTime: Instant,
-    val projects: List<SimpleProjectModel> = emptyList(),
+    val projects: List<ProjectModel<ProjectId>> = emptyList(),
```

Comment:
Maybe slightly clearer to use the `ExistingProjectModel` type alias instead of `ProjectModel<ProjectId>` here.